modules/machineset-yaml-rhv.adoc

```diff
@@ -65,6 +65,13 @@ spec:
             - compute <24>
           userDataSecret:
             name: worker-user-data
+spec:
```

Comment:
This is probably more of a question for the SME and QE reviewers, but I'll mention it here for reference. Is this YAML configuration supposed to have more than one top level `spec` and `spec: template` sections?

Reply:
You are right, it should be included in spec before, not be an entirely new spec section. 

So lines 68-74 should be deleted and the example should look something like this. It has to live under that providerSpec.value, order doesn't really matter, but the indentation does. 

```
... omitted .... 
      providerSpec:
        value:
          apiVersion: ovirtproviderconfig.machine.openshift.io/v1beta1
          cluster_id: <ovirt_cluster_id> <6>
          template_name: <ovirt_template_name> <7>
          instance_type_id: <instance_type_id> <8>
          **sparse: <boolean value> 
          format: <raw or cow>** 
          cpu: <9>
            sockets: <number_of_sockets> <10>
            cores: <number_of_cores> <11>
            threads: <number_of_threads> <12>
          memory_mb: <memory_size> <13>
          guaranteed_memory_mb:  <memory_size> <14>
          os_disk: <15>
            size_gb: <disk_size> <16>
            storage_domain_id: <storage_domain_UUID> <17>
          network_interfaces: <18>
            vnic_profile_id:  <vnic_profile_id> <19>
          credentialsSecret:
            name: ovirt-credentials <20>
          kind: OvirtMachineProviderSpec
          type: <workload_type> <21>
          auto_pinning_policy: <auto_pinning_policy> <22>
          hugepages: <hugepages> <23>
          affinityGroupsNames:
            - compute <24>
          userDataSecret:
            name: worker-user-data

```